lm_eval/__main__.py

```diff
@@ -292,13 +292,6 @@ def cli_evaluate(args: Union[argparse.Namespace, None] = None) -> None:
             "If fewshot_as_multiturn is set, apply_chat_template must be set to True."
         )
 
-    if (
-        args.num_fewshot is None or args.num_fewshot == 0
-    ) and args.fewshot_as_multiturn:
```

Comment:
the fewshot number can be defined in the task config file

Reply:
SGTM, if we are indeed 0-shot then `fewshot_as_multiturn` simply has no effect, correct?

Reply:
Yes, it should have no effect, resolving